path/resolver.go

```diff
@@ -129,8 +128,6 @@ func (s *Resolver) ResolveLinks(ctx context.Context, ndd *merkledag.Node, names
 
 		if nlink.Node == nil {
 			// fetch object for link and assign to nd
-			ctx, cancel := context.WithTimeout(ctx, time.Minute)
-			defer cancel()
```

Comment:
This should be here, or we should use a `fracctx` -- https://godoc.org/github.com/jbenet/go-context/frac